homeworks/vulmanjanos/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,38 @@
+from pprint import pprint
+
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": [
+        "pizza",
+        "carbonara",
+        "sushi"
+    ],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000"
+    }
+}
+
+program_languages = input("Mike, please give me 4 different program languages, seprated by a comma!")
+print(program_languages)
+program_language_list = program_languages.split(",") # listává konvertálás és elválasztás vesszővel
+user_info.update({"skills": program_language_list}) # lista hozzáadása user infóhoz
+user_info["favourite_meals"].sort() #lista abc sorrendbe rakása
+pprint(user_info)
+pprint(user_info["favourite_meals"][-2]) #az updatelt listáról az utolsó előtti elem kiírása
+user_info["favourite_meals"].append("spagetthi") # hozzáadom a listához a spagetthit
+pprint(user_info["favourite_meals"])
+user_info["favourite_meals"].extend(["pizza", "sushi"]) # hozzáadom újra a pizzát és a sushit
```

Comment:
ezt úgy hívjuk, hogy hard-coded változtatás. A megoldással az a baj, hogy ha esetleg más kaják kerülnek egy változtatás során a listába, akkor ez nem a 3. és 4. elemet adja majd hozzá az aktuális listából, hanem mindenféleképpen a pizzát és a sushit akkor is, ha amúgy azok benne sincsenek a listában.
Szuper az extend() használata, odáig jó. Viszont utána az értékek helyett próbálj az index-ekre hivatkozni. Tanultunk list slicingról (user_info["favourite_meals"][2:4]) ;)